Extension/src/cppTools1.ts

```diff
@@ -64,6 +64,7 @@ export class CppTools1 implements CppToolsTestApi, CppToolsTestExtension {
     }
 }
 
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
If we don't need `NullCppTools` anymore, just delete it.  It used to be returned during activation if cpptools was unable to run on that platform, but I see that code is gone now.

Reply:
The platform compatibility check is done in `checkVsixCompatibility`, but it appears that NullCppTools didn't come along for the ride.

Reply:
removed.